sharding/indexHashedNodesCoordinator.go

```diff
@@ -37,6 +38,8 @@ func NewIndexHashedNodesCoordinator(arguments ArgNodesCoordinator) (*indexHashed
 		selfPubKey:              arguments.SelfPublicKey,
 	}
 
+	ihgs.doExpandEligibleList = ihgs.expandEligibleList
```

Comment:
this is not needed.

Reply:
it is used in the nodesCoordinatorWithRater. Couldn't come up with a better idea and not change the indexHashedNodesCoordinator